packages/node_modules/@node-red/editor-client/src/sass/panels.scss

```diff
@@ -40,7 +40,7 @@
     height: 7px;
     box-sizing: border-box;
     cursor: ns-resize;
-    background: $primary-background url(images/grip.png) no-repeat 50% 50%;
+    background: $primary-background url(images/grip-horizontal.png) no-repeat 50% 50%;
```

Comment:
You will need to change the `.red-ui-panels-separator` definition on line 63 as well - that is for horizontal panels, which *should* use the existing `grip.png` background.

Reply:
Oops, my bad. :facepalm: